File: frontend/src/models/AssetDto.js
export default class AssetDto{
    constructor(
        id,
        code,
        name,
        brand,
        invoiceDate,
        price,
        currency,
        invoiceNumber,
        customFieldData
    ){
        this.id = id;
        this.code = code;
        this.name = name;
        this.brand = brand;
        this.invoiceDate = invoiceDate;
        this.price = price;
        this.currency = currency;
        this.invoiceNumber = invoiceNumber;
        this.customFieldData = customFieldData;
    }

}

File: backend/src/main/kotlin/com/worksap/assetwizard/vo/AssetDto.kt
package com.worksap.assetwizard.vo

import java.math.BigDecimal
import java.sql.Timestamp

data class AssetDto constructor(var id:Int,
                                var code:String,
                                var name:String,
                                var brand:String?,
                                var invoiceDate:Timestamp?,
                                var price:BigDecimal?,
                                var currency:String?,
                                var invoiceNumber:String?,
                                var customFieldData:List<AssetCustomColsDto>?
                                )